src/Generators/Microsoft.Gen.AutoClient/Microsoft.Gen.AutoClient.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RootNamespace>Microsoft.Gen.AutoClient</RootNamespace>
+    <Description>Code generator to support Microsoft.Extensions.Http.AutoClient.</Description>
+    <Workstream>Fundamentals</Workstream>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AnalyzerLanguage>cs</AnalyzerLanguage>
```

Comment:
Are we applies this property explicitly to all the analyzer projects? If so, consider declaring it centrally in one of the Directory.Build.props.